packages/flame_bloc/test/src/flame_bloc_listenable_test.dart

```diff
@@ -78,7 +92,7 @@ void main() {
 
         bloc.makeSad();
         await Future.microtask(() {});
-        expect(component.last, isNull);
+        expect(component.last, PlayerState.alive);
```

Comment:
```suggestion
        expect(component.last, equals(PlayerState.alive));
```

The same suggestion a couple of line bellow

Reply:
what's the difference?

Reply:
Matchers give better error message when a test fails